onevizion/integration/log.py

```diff
@@ -1,55 +0,0 @@
-import requests
-import json
-from onevizion.util import *
-from onevizion.curl import curl
-from onevizion.integration.loglevel import LogLevel
-from onevizion.httpbearer import HTTPBearerAuth
-import onevizion
-
-class IntegrationLog(object):
```

Comment:
I think we should not remove old classes right now and we should mark them as deprecated. But old classes should also use new API endpoints.

Reply:
I did not delete the old classes, but renamed the **integration** directory to **module**, because it won't affect anything
Please pay attention to the **onevizion/module/log.py** file, there is still an **IntegrationLog** class there

Reply:
I see